src/pybind/mgr/balancer/module.py

```diff
@@ -189,6 +191,8 @@ def calc_stats(self, count, target, total):
             stddev = math.sqrt(dev / float(max(num - 1, 1)))
             score = score / max(sum_weight, 1)
             r[t] = {
+                'max': max(count[t].itervalues()),
+                'min': min(count[t].itervalues()),
```

Comment:
`itervalues()` is not compatible with py3.

Reply:
@yanghonggang mind switching to  `values()`? then this should be good to go